app/models/accounting/quickbooks/updater.rb

```diff
@@ -0,0 +1,77 @@
+module Accounting
+  module Quickbooks
+    class FullSyncRequiredError < StandardError; end
+
+    class Updater
```

Comment:
Really need a comment on this class. Please try to put comments on all classes. What does it do? How is it different than FullFetcher? FullFetcher needs a comment too please.